Client/Client.Qt/registration.cpp

```diff
@@ -68,6 +68,8 @@ Registration::Registration(QWidget* parent) : QWidget(parent)
         }
 
         ConnectionManager::getClient().userRegistration(email, login, password);
+
+        oApp->setAppState(App::AppState::LoginForm);
```

Comment:
What if server sends a failed result?

Reply:
I mean the name of task https://github.com/f1nal3/Juniorgram/issues/329 is on **Successful registration** but something can happen so it is not.